packages/@ember/-internals/glimmer/lib/resolver.ts

```diff
@@ -67,6 +68,11 @@ function layoutFor(
 ): Nullable<Template> {
   let templateFullName = `template:components/${name}` as const;
 
+  deprecateUntil(
```

Comment:
It looks like this will fire a deprecation if somebody has a JS with no HBS at all?

I think we would only want to fire the deprecation if `layoutFor` successfully finds a result.

Reply:
fixed! thanks!